### PR TITLE
ShareAccount

### PR DESCRIPTION
**Pull request by Sasha-Egorov**

### CODE REVIEW

File: src/classes/ShareAccountController.cls [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Created by Synebo on 12.10.2021.
+ */
+
+public with sharing class ShareAccountController {
+    @AuraEnabled
+    public static void createAccountShare(Id recordId, Id userOrGroupId) {
+        AccountShare Acctshr = new AccountShare();
```
Comment (ShYevhen) [inline src/classes/ShareAccountController.cls]: Will we have 2 AccountShare records if will share with one user twice?

File: src/lwc/lwc_QuickAction/lwc_QuickAction.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    handleChange(event){
+        this.userOrGroupId = event.target.value;
+    }
+
+    closeAction(){
+        this.dispatchEvent(new CloseActionScreenEvent());
+    }
+    Save(){
+        console.log(this.recordId);
```
Comment (ShYevhen) [inline src/lwc/lwc_QuickAction/lwc_QuickAction.js]: in real code all not necessary console.log or system.debug should be removed after development is done.

File: src/classes/ShareAccountController.cls [hunk 3]
```diff
@@ -0,0 +1,19 @@
+public with sharing class ShareAccountController {
+    @AuraEnabled
+    public static void createAccountShare(Id recordId, Id userOrGroupId) {
+        List<AccountShare> checkForAvailability = [
+                SELECT Id, AccountId,UserOrGroupId
+                FROM AccountShare
+                WHERE AccountId = :recordId
+                AND UserOrGroupId = :userOrGroupId
+        ];
+        if (checkForAvailability.isEmpty()) {
```
Comment (EvgeniyShvorakOH) [inline src/classes/ShareAccountController.cls:10]: if 'checkForAvailability' not empty, can be checked AccessLevel to objects and updated if record has less access then you provide.

Comment (ShYevhen) [inline src/classes/ShareAccountController.cls:10]: if 'checkForAvailability' not empty, can be checked AccessLevel to objects and updated if record has less access then you provide.

**Pull request merged**